staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -426,3 +434,29 @@ func getEnvelopePrefixTransformer(config *apiserverconfig.KMSConfiguration, enve
 		Prefix:      []byte(prefix + config.Name + ":"),
 	}, nil
 }
+
+// parseEndpoint parses the endpoint to extract schema, host or path.
+func parseEndpoint(endpoint string) (string, error) {
+	if len(endpoint) == 0 {
+		return "", fmt.Errorf("remote KMS provider can't use empty string as endpoint")
```

Comment:
Be more consistent with form of other errors, e.g. "invalid empty Resources[].KMS.Endpoint in encryption configuration..." ?